Source/WebCore/css/ComputedStyleExtractor.cpp

```diff
@@ -2927,8 +2932,10 @@ static Ref<CSSValue> valueForOffsetShorthand(const RenderStyle& style)
     bool nonInitialDistance = !style.offsetDistance().isZero();
     bool nonInitialRotate = style.offsetRotate() != style.initialOffsetRotate();
 
-    if (style.offsetPath() || nonInitialDistance || nonInitialRotate)
-        innerList.append(valueForPathOperation(style, style.offsetPath(), SVGPathConversion::ForceAbsolute));
+    if (style.offsetPath() || nonInitialDistance || nonInitialRotate) {
+        if (auto pathOperation = valueForPathOperation(style, style.offsetPath(), SVGPathConversion::ForceAbsolute))
+            innerList.append(*pathOperation);
```

Comment:
Should this whole function return null if pathOperation is null, rather than just skipping serializing that value?